index.css

```diff
@@ -37,13 +42,15 @@ img {
   z-index: 100;
 }
 
-#feedback-text {
+/* #feedback-text {
   position: absolute;
   left: 50%;
-  top: 0%;
-  z-index: 0;
+  top: 21%;
   transform: translate(-50%, 0%);
-}
+  height: 50px;
+  width: 100%;
+  border: 1px solid red;
+} */
```

Comment:
Do we still need these css styling?

Reply:
@bernhardccodev i removed this one. thanks. please check